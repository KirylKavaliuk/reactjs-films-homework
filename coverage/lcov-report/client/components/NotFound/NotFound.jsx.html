<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for client/components/NotFound/NotFound.jsx</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="..\..\..\prettify.css" />
    <link rel="stylesheet" href="..\..\..\base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(..\..\..\sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="..\..\..\index.html">All files</a> / <a href="index.html">client/components/NotFound</a> NotFound.jsx
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>2/2</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>1/1</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>2/2</span>
      </div>
    </div>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet">1
2
3
4
5
6
7
8
9
10
11
12
13
14
15
16
17
18</td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">1x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React, { Component } from 'react';
&nbsp;
import Link from 'components/Link/Link';
&nbsp;
import styles from './NotFound.scss';
&nbsp;
const NotFound = () =&gt; (
    &lt;div className={ styles.notFound }&gt;
      &lt;div className={ styles.text }&gt;
        &lt;h1 className={ styles.header }&gt;Error.&lt;/h1&gt;
        &lt;p className={ styles.description }&gt;items is not found&lt;/p&gt;
        &lt;Link to='/' className={ styles.link }&gt;To tranding movies&lt;/Link&gt;
      &lt;/div&gt;
    &lt;/div&gt;
);
&nbsp;
export default NotFound;
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Fri Feb 01 2019 11:31:09 GMT+0300 (GMT+03:00)
</div>
</div>
<script src="..\..\..\prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="..\..\..\sorter.js"></script>
</body>
</html>
