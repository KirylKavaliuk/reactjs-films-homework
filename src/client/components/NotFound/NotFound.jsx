import React from 'react';

import Link from 'components/Link/Link';

import styles from './NotFound.scss';

const NotFound = () => (
  <div className={ styles.notFound }>
    <div className={ styles.text }>
      <h1 className={ styles.header }>Error.</h1>
      <p className={ styles.description }>items is not found</p>
      <Link to='/trading' className={ styles.link }>To tranding movies</Link>
    </div>
  </div>
);

export default NotFound;
