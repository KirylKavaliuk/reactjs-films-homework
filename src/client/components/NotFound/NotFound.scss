@import 'styles/palette';

.not-found {
  display: flex;
  flex-direction: column;
  min-height: 20vh;
  align-items: center;
  justify-content: space-between;
}

.text {
  margin: 30vh 0;
  text-align: center;
}

.header {
  color: $LIGHT_BLUE;
  font-size: 40px;
  font-weight: 700;
}

.description {
  color: $GREY;
  font-size: 18px;
}

.link {
  color: $DARK_GREY;
  font-size: 16px;
  padding: 20px 0;
  box-sizing: border-box;
  display: block;
  text-decoration: underline;
}
