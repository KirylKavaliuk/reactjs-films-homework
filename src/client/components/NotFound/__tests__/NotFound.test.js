import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import NotFound from '../NotFound';

it('renders not found component correctly', () => {
  const renderer = new ShallowRenderer();
  renderer.render(<NotFound/>);

  const tree = renderer.getRenderOutput();

  expect(tree).toMatchSnapshot();
});
