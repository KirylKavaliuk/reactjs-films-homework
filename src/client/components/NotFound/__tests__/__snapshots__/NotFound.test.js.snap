// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NotFound/> 1`] = `
<div
  className="NotFound__not-found___vgFr0"
>
  <div
    className="NotFound__text___vJrW1"
  >
    <h1
      className="NotFound__header___3ES7q"
    >
      Error.
    </h1>
    <p
      className="NotFound__description___39rly"
    >
      items is not found
    </p>
    <_Link
      className="NotFound__link___3l-Er"
      clearParams={Array []}
      params={Object {}}
      to="/"
    >
      To tranding movies
    </_Link>
  </div>
</div>
`;
